pds4indextools/pds4_create_xml_index.py

```diff
@@ -46,14 +46,46 @@
 """
 
 import argparse
+from collections import namedtuple
 import configparser
+from itertools import groupby
 from lxml import etree
 import pandas as pd
 from pathlib import Path
 import requests
 import sys
 
 
+SplitXPath = namedtuple('SplitXPath',
+                        ['xpath', 'parent', 'child', 'prefix', 'num'])
+
+
+def convert_header_to_xpath(root, xpath_find, namespaces):
+    """Convert an XML header path to an XPath expression.
```

Comment:
I have no idea what this function is doing. Perhaps some more explanatory text would help someone understand the purpose? See my renumber function (the original version) for an example of giving details.